src/components/navigation/breadcrumb/Breadcrumb.js

```diff
@@ -16,10 +16,21 @@ function Breadcrumb({ children, ...props }) {
 }
 
 Breadcrumb.propTypes = {
-  children: PropTypes.oneOfType([
-    PropTypes.element,
-    PropTypes.arrayOf(PropTypes.element)
-  ])
+  children: (props, propName, component) => {
+    if (
+      PropTypes.oneOfType([
+        PropTypes.element,
+        PropTypes.arrayOf(PropTypes.element)
+      ])

```

Comment:
I think you may have the logic in this case backwards. Normally the proptype functions return undefined or null if  successful and an error object is there is a problem. I'm pretty sure we'd go into this if block if we get an error back from `oneOfType()`, which I don't think we want.

Reply:
Is the required name attribute of children just used for the key?  I'd rather see a programmatic way of generating the key rather than relying on an attribute that is a little confusing and restricts what this can use as children.